stylesheets/jbt-styles.scss

```diff
@@ -403,41 +417,41 @@ kbd {
 .videoblock {
   margin-top: 1em;
   margin-bottom: 1em;
-  
+
   iframe {
     margin-left: auto;
     margin-right: auto;
     display: block;
   }
-  
+
   @media (min-width: 981px) {
```

Comment:
all those changes occurred when I edited this file in Atom. I can revert those changes if needed